source/appModules/winword.py

```diff
@@ -54,7 +54,9 @@ class WinwordWordDocument(WordDocument):
 	def _get_description(self) -> str:
 		try:
 			curView = self.WinwordWindowObject.view.Type
-			return f"{ViewType(curView).displayString} {super()._get_description()}"
+			if isinstance(super()._get_description(), str):
```

Comment:
I'd rather check the Truthiness of description, since that ensures empty strings are also discarded properly.
```suggestion
			if description:
```

Reply:
Thanks @LeonarddeR . I thought about doing this before sending this PR, but this didn't work on my side, so I used isinstance. I'll try again.

Reply:
Actually, this needs to be something like this:
```suggestion
			if isinstance(description, str):
```

Please ensure you are using description everywhere super()._get_description() was used first.